packages/vue-components/package.json

```diff
@@ -32,5 +32,12 @@
     "@vue/cli-plugin-unit-jest": "4.4.4",
     "@vue/cli-service": "4.4.4",
     "@vue/test-utils": "1.1.1"
+  },
+  "peerDependencies": {
+    "vue2mapbox-gl": "^0.13.0"
+  },
+  "dependencies": {
+    "@deltares/utilities": "^0.2.0",
```

Comment:
@sjoerdbeentjes @petergoes what do you guys think about this as a regular dependency?

Reply:
That sounds logical to me 👍